mut/index/test_SnootyManifest.py

```diff
@@ -0,0 +1,122 @@
+from base64 import decode
+from pydoc import Doc, doc
+from bson import decode_all
+from json import loads
+from pathlib import Path
+
+from mut.index.SnootyManifest import Document, generate_manifest
+
+
+ROOT_PATH = Path("/Users/allison/snooty/mut/mut/test_data_index/documents")
```

Comment:
Clearly I'll need to fix this.

Reply:
Question mark :D

Reply:
So realtalk, I'm not really sure *how* to. Will ping you 😁